estore_project/users/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import path
+from rest_framework_simplejwt.views import TokenRefreshView
 
-from .views import Login, Signup
+from .views import LoginView, Signup
 
 app_name = "users"
 urlpatterns = [
     path("signup/", view=Signup.as_view(), name="signup"),
-    path("login/", view=Login.as_view(), name="login"),
+    path("login/", LoginView.as_view(), name="login"),
```

Comment:
why not do  `path("login/", TokenObtainPairView.as_view(), name="login")` and remove the `LoginView`

Reply:
I need to get user_id in response as well along with the access and refresh tokens so I wrote my own serializer and mentioned that serializer in LoginView.